pkg/systemd/generate/containers.go

```diff
@@ -69,6 +69,9 @@ type containerInfo struct {
 	ExtraEnvs []string
 	// EnvVariable is generate.EnvVariable and must not be set.
 	EnvVariable string
+	// OptEnvs contains the container environment variables setted by
+	// -e/--env options.
```

Comment:
I suggest removing the reference on the CLI flag as it's not important in/for this package.
```suggestion
	// -e/--env options.
```